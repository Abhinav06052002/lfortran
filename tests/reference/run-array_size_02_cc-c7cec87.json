{
    "basename": "run-array_size_02_cc-c7cec87",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/errors/array_size_02_cc.f90",
    "infile_hash": "19e1731eec10f3e78ca24c5ce8d7427a8da4f1ed7148ceadaba7c64e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-array_size_02_cc-c7cec87.stdout",
    "stdout_hash": "a779ee4f7341ddce58a41cee3d249ecf1b37216d7d4f16eb4e0f528e",
    "stderr": "run-array_size_02_cc-c7cec87.stderr",
    "stderr_hash": "55e22f8eef5113beee02dafb996968cfc6316a6803eb1ddfff619f1b",
    "returncode": 1
}