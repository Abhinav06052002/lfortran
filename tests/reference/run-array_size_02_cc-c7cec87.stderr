semantic error: dim has already been specified as a positional/keyword argument to size
 --> tests/errors/array_size_02_cc.f90:5:14
  |
5 |     size_a = size(a, 1, dim=1)
  |              ^^^^^^^^^^^^^^^^^ 

semantic error: dim has already been specified as a positional/keyword argument to size
 --> tests/errors/array_size_02_cc.f90:6:14
  |
6 |     size_a = size(a, dim = 1, 1)
  |              ^^^^^^^^^^^^^^^^^^^ 
